noxfile.py

```diff
@@ -14,19 +14,19 @@
 
 import nox
 
-@nox.session(python='3.6')
+@nox.session(python=['3.6', '3.8'])
```

Comment:
Out of curiosity do we want `'3.7'` as well?

Reply:
For completeness, perhaps.  But the additional cost would be it takes longer for CI/CD to complete for each pull request, and I don't think it's going to catch much given we're checking 3.6 and 3.8.